.github/renovate.json5

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
This should be somehow similar to what we do with Dependabot: https://github.com/adeira/universe/blob/3b3e1e4bca7a63262b1dcba2187be9e003dbeb6a/.github/dependabot.yml